src/lib/steps.ts

```diff
@@ -72,6 +72,12 @@ export interface SortStep {
   order?: Array<'asc' | 'desc'>;
 }
 
+export interface FillnaStep {
+  name: 'fillna';
+  column: string;
+  value: string | number;
+}
```

Comment:
What about deifning a new type for such cases, like:

```
type primitiveType = number | boolean | string | Date;
```

Reply:
Done, thank you :)